doc/manual/manual.bib

```diff
@@ -12,6 +12,72 @@ @string{ajs
 @string{cmame = {Computer Methods in Applied Mechanics and Engineering}}
 @string{pnas = {Proceedings of the National Academy of Sciences}}
 
+@article{burb96,
+title={{Effect of depth-dependent viscosity on the planform of mantle convection}},
+author={H.-P. Bunge and M.A. Richards and J.R. Baumgardner},
+journal={Nature},
+volume={379},
+number={},
+pages={436--438},
+year={1996}}
+
+@article{youn74,
+title={Finite-amplitude thermal convection in a spherical shell},
+author={Young, Richard E},
+journal={J. Fluid Mech.},
+volume={63},
+number={4},
+pages={695--721},
+year={1974}}
+
+@article{buss75,
+title={{Patterns of convection in spherical shells}},
+author={F.H. Busse},
+journal={J. Fluid Mech.},
+volume={72},
+number={1},
+pages={67--85},
+year={1975}}
+
+@article{buss79,
+title={High Prandtl number convection},
+author={Busse, FH},
+journal=pepi,
+volume={19},
+number={2},
+pages={149--157},
+year={1979}}
+
+@article{blbc89,
+title={{A benchmark comparison for mantle convection codes}},
+author={B. Blankenbach and F. Busse and U. Christensen and L. Cserepes and D. Gunkel
+and U. Hansen and H. Harder and G. Jarvis and M. Koch and G. Marquart and D. Moore
+and P. Olson and H. Schmeling and T. Schnaubelt},
+journal=gji,
+volume={98},
+pages={23-38},
+year={1989}}
+
+@article{vavy93,
+title={The effects of a composite non-Newtonian and Newtonian rheology on mantle convection},
```

Comment:
```suggestion
title={The effects of a composite non-{N}ewtonian and {N}ewtonian rheology on mantle convection},
```

Reply:
@cedrict did you see Wolfgang's comments? I will stop here for now.